src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/ConfigPersistenceManager.java

```diff
@@ -22,217 +24,248 @@
+    }
+
+    public static boolean isKnowledgeTypeEnabled(String projectKey, String knowledgeType) {
+	String isKnowledgeTypeEnabled = getValue(projectKey, knowledgeType);
+	return "true".equals(isKnowledgeTypeEnabled);
+    }
+
+    public static boolean isUseClassiferForIssueComments(String projectKey) {
+	return getValue(projectKey, "setClassiferForIssueComments").equals("true");
+    }
+
+    public static boolean isWebhookEnabled(String projectKey) {
+	String isWebhookEnabled = getValue(projectKey, "isWebhookEnabled");
+	return "true".equals(isWebhookEnabled);
+    }
+
+    public static boolean isWebhookTypeEnabled(String projectKey, String webhookType) {
+	if (webhookType == null || webhookType.isBlank()) {
+	    return false;
+	}
+	String isWebhookTypeEnabled = getValue(projectKey, "webhookType" + "." + webhookType);
+	return "true".equals(isWebhookTypeEnabled);
+    }
+
+    public static void setActivated(String projectKey, boolean isActivated) {
+	setValue(projectKey, "isActivated", Boolean.toString(isActivated));
+    }
+
+    public static void setIconParsing(String projectKey, boolean isIconParsing) {
+	setValue(projectKey, "isIconParsing", Boolean.toString(isIconParsing));
+    }
+
+    public static void setIssueStrategy(String projectKey, boolean isIssueStrategy) {
+	setValue(projectKey, "isIssueStrategy", Boolean.toString(isIssueStrategy));
+    }
+
+    public static void setKnowledgeExtractedFromGit(String projectKey, boolean isKnowledgeExtractedFromGit) {
+	setValue(projectKey, "isKnowledgeExtractedFromGit", Boolean.toString(isKnowledgeExtractedFromGit));
+	if (isKnowledgeExtractedFromGit) {
+	    new GitClientImpl(projectKey);
+	}
+    }
+
+    // TODO Testing
+    public static void setPostSquashedCommits(String projectKey, Boolean checked) {
+	setValue(projectKey, "isPostSquashedCommitsActivated", Boolean.toString(checked));
+    }
+
+    // TODO Testing
+    public static void setPostFeatureBranchCommits(String projectKey, Boolean checked) {
+	setValue(projectKey, "isPostFeatureBranchCommitsActivated", Boolean.toString(checked));
+    }
+
+    public static void setGitUris(String projectKey, String gitUris) {
+	setValue(projectKey, "gitUris", gitUris);
+    }
+
+    public static List<String> getGitUris(String projectKey) {
+	String value = getValue(projectKey, "gitUris");
+	List<String> uris = Arrays.asList(value.split(";;"));
+	return uris;
+    }
+
+    public static void setDefaultBranches(String projectKey, String defaultBranches) {
+	setValue(projectKey, "defaultBranches", defaultBranches);
+    }
+
+    public static Map<String, String> getDefaultBranches(String projectKey) {
+	Map<String, String> defaultBranches = new HashMap<String, String>();
+	String value = getValue(projectKey, "gitUris");
+	List<String> uris = Arrays.asList(value.split(";;"));
+	if (value == null || value.equals("")) {
+	    return null;
+	}
+	value = getValue(projectKey, "defaultBranches");
+	if (value.equals(null) || value.equals("")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using equals() to compare against null](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4916975)